tutorial/tests/test_basic_datatypes.py

```diff
@@ -203,15 +203,17 @@ def test_remove_every_second_element_from_list(my_list, function_to_test):
     )
 
 
-def reference_return_first_and_last_element_from_list(my_list: list) -> tuple:
-    return my_list[0], my_list[-1]
+def reference_return_first_and_last_element_from_list(my_list: list) -> list:
```

Comment:
`solution_return_first_and_last_element_from_list` is still expecting a `tuple[float]` as a result

Reply:
Hmm i think i want the tuple or not? returning the furst and last element with a comma will give a tuple. I think tuple is more consistent with the return format of the rest of the notebooks, or should we put list? then we should also change the return of the function.

Reply:
Yes, let's keep the tuple and make them consistent

Reply:
> Yes, let's keep the tuple and make them consistent

Once we also address this, I think we are good to go 👍